src/helpers/getPropOr.js

```diff
@@ -0,0 +1,38 @@
+/** @license ISC License (c) copyright 2019 original and current authors */
+/** @author Ian Hofmann-Hicks */
+
+const curry = require('../core/curry')
+const isDefined = require('../core/isDefined')
+const isEmpty = require('../core/isEmpty')
+const isNil = require('../core/isNil')
+const isInteger = require('../core/isInteger')
```

Comment:
Does the order matter with imports in code or only in docs?

Reply:
It matters everywhere! great 👁 will fix.